### DISCUSSION TITLE
Language and RTL Support in the Template

### DISCUSSION THREAD

**Discussion by rysecurity** [alshedivat/al-folio#2609] [Q&A] [answered]
I’m interested in adding a language other than English to the template. Could you guide me on how to do this?

Additionally, does the template support right-to-left (RTL) text, as Arabic is read from right to left? If not, do you have any suggestions? 
I’ve tried a few approaches, but the page layout becomes corrupted because the content shifts to the right.

Thank you in advance

**Comment by george-gca** [comment, marked as answer]
About other languages, check #1721.

About RTL, a PR (#946) was created for this a long time ago, but I have no idea about how it was back then.
